### PR TITLE
Add works with page

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Add works with page. This also fixes highlights of links.

#### Screenshots

Screen Shot 2023-09-08 at 10 04 29

#### Changes

- Add works with page
- Add CSS to display smaller non-float logos.

#### Notes for Reviewers

- I didn't use a more generic "The Things Stack Certification" heading since that is very similar to [The Things Stack Certified](https://www.thethingsnetwork.org/achievements/a/the-things-stack-certified/). 

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/reference/works-with/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Works with The Things Stack"
+description: ""
+---
+
+Selected LoRaWAN End Devices and Gateways are certified **Works with The Things Stack**.
+
+<!--more-->
+
+{{< figure src="./works-with-tts-light.png" alt="Works wit The Things Stack" class="logo">}}
```
Comment (johanstokking) [inline doc/content/reference/works-with/_index.md:10]:
```suggestion
{{< figure src="./works-with-tts-light.png" alt="Works with The Things Stack" class="logo">}}
```

**Review by johanstokking — APPROVED** [review]
It looks a bit like a loose page in reference... should it be under end devices or closer to TTS?

**Comment by KrishnaIyer** [comment]
This covers both devices and gateways. Should I just create a separate page for devices and gateways under their respective sections?

**Comment by johanstokking** [comment]
Then maybe under The Things Stack section?

Reference should really be for more detailed stuff. This is more like an overview explanation page that is high level.

**Comment by KrishnaIyer** [comment]
Screen Shot 2023-09-11 at 09 34 14
Ok that works indeed.

**Review by johanstokking — APPROVED** [review]

**Pull request merged**